api.ts

```diff
@@ -1372,6 +1376,24 @@ export interface DynamicSyncTableOptions<
    * this sync table (including autocomplete formulas).
    */
   connectionRequirement?: ConnectionRequirement;
+  /**
+   * If true, when subsequent syncs discover new schema fields, these fields will not automatically
```

Comment:
you are missing a "be" at the end of the line

Reply:
good catch!